test/edu-benefits/5490/schema/schema.unit.spec.js

```diff
@@ -26,5 +27,13 @@ describe('5490 schema tests', () => {
         }
         expect(result.valid).to.be.true;
       });
+      // Expect the data to be what's found in expected-data/${file}
+      if (expectedDataFiles.indexOf(file) !== -1) {
```

Comment:
Should probably add a comment here to replace this with includes() once the version of Node.js on Jenkins is upgraded